### PR TITLE
Update references to legacy images 

### PR DESCRIPTION
**Pull request by kbatuigas**
# Description
Updated legacy image references to the corresponding next-gen images (generally, `circleci` -> `cimg`).

More references will be updated as more next-gen images are released. For some I'm still waiting on clarification as to whether they need more than just the name swap.

# Reasons
Legacy images will be deprecated this year on 12/31. Some of these legacy images already have their next-gen counterparts; others are still being built.

### CODE REVIEW

**Comment by kbatuigas** [comment]
> @kbatuigas Can you please remove your changes to the `package-lock.json`? Content changes should not trigger changes to dependencies. Thanks!

Hi @rsylvian , I wasn't sure how to trace back the changes I made to `package-lock.json`, but I figured I'd just have you re-review in case that's now been resolved.

**Comment by rsylvian** [comment]
Thanks, @kbatuigas!

**Review by rsylvian — APPROVED** [review]
Approving because I originally requested changes about the package-lock. I'm not qualified to approve the actual content changes so feel free to ask other people on that matter 🙌

File: jekyll/_cci2/code-coverage.md [hunk 1]
```diff
@@ -64,7 +64,92 @@ end
...
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+        environment:
+          POSTGRES_USER: circleci-demo-ruby
+          POSTGRES_DB: rails_blog
+          POSTGRES_PASSWORD: ""
+    steps:
+      - checkout
+      - run:
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:91]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/code-coverage.md [hunk 2]
```diff
@@ -64,7 +64,92 @@ end
...
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:131]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/code-coverage.md [hunk 3]
```diff
@@ -135,7 +220,80 @@ coverage html  # open htmlcov/index.html in a browser
...
+jobs:
+  build:
+    docker:
+    - image: cimg/python:3.10-browsers
+      auth:
+        username: mydockerhub-user
+        password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+    - checkout
+    - run:
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:237]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/code-coverage.md [hunk 4]
```diff
@@ -135,7 +220,80 @@ coverage html  # open htmlcov/index.html in a browser
...
+        name: Run Tests
+        command: |
+          $HOME/.local/bin/coverage run -m pytest
+          $HOME/.local/bin/coverage report
+          $HOME/.local/bin/coverage html  # open htmlcov/index.html in a browser
+    - store_artifacts:
+        path: htmlcov
+workflows:
+  version: 2
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:251]: This line isn't needed with config v2.1.

File: jekyll/_cci2/code-coverage.md [hunk 5]
```diff
@@ -135,7 +220,80 @@ coverage html  # open htmlcov/index.html in a browser
...
+jobs:
+  build:
+    docker:
+    - image: cimg/python:3.10-browsers
+      auth:
+        username: mydockerhub-user
+        password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+    - checkout
+    - run:
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:271]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/code-coverage.md [hunk 6]
```diff
@@ -135,7 +220,80 @@ coverage html  # open htmlcov/index.html in a browser
...
+        name: Run Tests
+        command: |
+          $HOME/.local/bin/coverage run -m pytest
+          $HOME/.local/bin/coverage report
+          $HOME/.local/bin/coverage html  # open htmlcov/index.html in a browser
+    - store_artifacts:
+        path: htmlcov
+workflows:
+  version: 2
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:285]: This line isn't needed in v2.1 config.

File: jekyll/_cci2/code-coverage.md [hunk 7]
```diff
@@ -253,9 +411,52 @@ also converted to an `html` page, like many other coverage tools. The Pom file
...
+jobs:
+  build:
+    docker:
+      - image: cimg/openjdk:17.0-browsers
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+      - checkout
+      - run : mvn test
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:430]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/code-coverage.md [hunk 8]
```diff
@@ -253,9 +411,52 @@ also converted to an `html` page, like many other coverage tools. The Pom file
...
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:449]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/code-coverage.md [hunk 9]
```diff
@@ -278,7 +479,57 @@ jobs:
...
+jobs:
+  build:
+    docker:
+      - image: cimg/node:17.2-browsers
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+      - checkout
+      - run: npm install
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:496]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/code-coverage.md [hunk 10]
```diff
@@ -278,7 +479,57 @@ jobs:
...
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:518]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/code-coverage.md [hunk 11]
```diff
@@ -308,8 +559,54 @@ a tool called phpdbg; you can generate a report using the command `phpdbg -qrr v
...
+jobs:
+  build:
+    docker:
+      - image: cimg/php:8.1-browsers
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+      - checkout
+      - run:
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:576]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/code-coverage.md [hunk 12]
```diff
@@ -308,8 +559,54 @@ a tool called phpdbg; you can generate a report using the command `phpdbg -qrr v
...
```
Comment (felicianotech) [inline jekyll/_cci2/code-coverage.md:597]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/config-intro.md [hunk 13]
```diff
@@ -111,11 +111,14 @@ Every code base and project is different.  That’s okay.  We like diversity.  T
...
+  browser-tools: circleci/browser-tools@1.1.0
 jobs:
   build:
     # pre-built images: https://circleci.com/docs/2.0/circleci-images/
     docker:
-      - image: circleci/node:14-browsers
+      - image: cimg/node:17.2-browsers
     steps:
       - checkout
       - run:
```
Comment (felicianotech) [inline jekyll/_cci2/config-intro.md:124]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

**Review by felicianotech — CHANGES_REQUESTED** [review]

**Review by felicianotech — APPROVED** [review]

**Pull request merged**